Project/Source/Modules/ModuleProject.cpp

```diff
@@ -341,6 +343,19 @@ bool ModuleProject::CleanUp() {
 	return true;
 }
 
+void ModuleProject::ReceiveEvent(TesseractEvent& e) {
+	for (ComponentScript& script : App->scene->scene->scriptComponents) {
+		if (App->time->HasGameStarted() && App->scene->sceneLoaded) {
+			if (script.IsActiveInHierarchy()) {
```

Comment:
This will conflict when merging develop

it now changed to  `script.IsActive()`